setup.py

```diff
@@ -88,7 +88,7 @@ def run(self):
     ],
     install_requires = [
         'traitlets',
-        'jupyter_core',
+        'jupyter_core>=4.5.0',
```

Comment:
Since we're already building a [`4.6.0` jupyter_core release](https://github.com/jupyter/jupyter_core/pull/164) - which is waiting on the final resolution for secure_write,  I think we should just move to 4.6.0 with this change.